android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java

```diff
@@ -597,6 +725,37 @@ public void reportEndCallWithUUID(String uuid, int reason) {
         conn.reportDisconnect(reason);
     }
 
+   @Override
+   public void onHostResume() {
+       Log.d(TAG, "onResume()");
+   }
+
+   @Override
+   public void onHostPause() {
+       Log.d(TAG, "onPause()");
+   }
+
+   @Override
+   public void onHostDestroy() {
+      // when activity destroyed end all calls
+       Log.d(TAG, "onDestroy()");
```

Comment:
`onHostDestroy`

Reply:
Whats wrong here?

Reply:
The log should be : `Log.d(TAG, "onHostDestroy()");` not `Log.d(TAG, "onDestroy()");`